terraform/main.tf

```diff
@@ -0,0 +1,94 @@
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_group_name
+  location = var.resource_group_location
+}
+
+
+# SQL Server
+resource "azurerm_mssql_server" "pc_component_gallery_server" {
+  name                          = var.sql_server_name
+  resource_group_name           = azurerm_resource_group.rg.name
+  location                      = azurerm_resource_group.rg.location
+  version                       = "12.0"
+  administrator_login           = var.sql_admin_username
+  administrator_login_password  = var.sql_admin_password
+  public_network_access_enabled = true
+}
+
+# SQL Server Firewall Rule to allow Azure services access
+resource "azurerm_mssql_firewall_rule" "allow_azure_services" {
+  name             = "AllowAzureServices"
+  server_id        = azurerm_mssql_server.pc_component_gallery_server.id
+  start_ip_address = "0.0.0.0"
```

Comment:
Jesteś tego pewien? To raczej zablokuje kompletnie możliwość podpięcia się do bazy

Reply:
według stacka to powinno zadziałać: https://stackoverflow.com/questions/75948700/how-to-set-allow-azure-services-and-resources-to-access-this-server-via-terraf

Ale trzeba przetestować

Reply:
![image](https://github.com/user-attachments/assets/f9a25762-e8a0-4548-b968-92ac64a44054)
działa